test/rootfs.jl

```diff
@@ -27,6 +27,7 @@ using BinaryBuilderBase
     ]
     @test expand_gfortran_versions([Platform("x86_64", "linux"; sanitize="memory")]) ==
         [Platform("x86_64", "linux"; sanitize="memory")]
+    @test expand_gfortran_versions(Platform[]) == Platform[]
```

Comment:
This isn't a good test:
```julia
julia> [] == Platform[]
true
```
You want to check either the `eltype` or, which is pretty much the same, `x isa Vector{Platform}`